tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -18,6 +18,7 @@
 #include "tt_metal/impl/dispatch/command_queue_interface.hpp"
 #include "tt_metal/impl/dispatch/device_command.hpp"
 #include "tt_metal/impl/dispatch/lock_free_queue.hpp"
+#include "tt_metal/impl/dispatch/data_collection.hpp"
```

Comment:
Should this be in the cpp instead since the hpp doesn't seem to need it?

Reply:
Yeah, moved it in latest commit